cmd/prune.go

```diff
@@ -31,26 +31,36 @@ import (
 	"github.com/dims/maintainers/pkg/utils"
 )
 
-var dryRun, skipGH, skipDS bool
-var repositoryDS, repositoryGH, periodDS string
-var includes, excludes []string
-var excludeFiles []string
+type option struct {
+	dryRun			bool
+	skipDS			bool
+	skipGH			bool
+	repositoryDS 	string
+	repositoryGH	string
+	periodDS		string
+	includes		[]string
+	excludes		[]string
+	excludeFiles	[]string
 
+}
```

Comment:
I think it's better to use plural for the name here. `option` -> `options`.

Also, can you please run `go fmt` on the code? Tiny nit: there's an extra new line and the type indentation seems to be off.